homeassistant/components/fjaraskupan/fan.py

```diff
@@ -78,7 +79,7 @@ def __init__(
         super().__init__(coordinator)
         self._device = device
         self._default_on_speed = 25
-        self._attr_name = device_info["name"]
+        self._attr_name = None
```

Comment:
This is the default, the line can be removed instead.

Reply:
```suggestion
```